### ISSUE TITLE
Forcibly resigned Tsar is then picked as Tsar for next round

### ISSUE THREAD

**Issue by domcleal** [grifferz/pah-irc#85]
johnm was the Tsar and due to pick the winning card:

`<+AgainstHumanity>` Waiting on johnm to pick the winning play. Round started about 10h16m ago. Idle punishment in about 13h45m.
`<+AgainstHumanity>` The Card Tsar is johnm; current Black Card:

He was then forcibly resigned while still waiting:

`<+AgainstHumanity>` I'm forcibly resigning johnm from the game due to idleness. Idle since 2015-02-26T11:55:58.

A new round started, but PAH picked him again as the Tsar:

`<+AgainstHumanity>` The Card Tsar is johnm; current Black Card:

My reading of the code is that he should have been marked inactive after the previous round, so shouldn't have been able to be picked as the new Tsar.

**Comment by grifferz** [comment]
Duplicate of #86.
